### PR TITLE
Public repos do not need PAT to access rest API

### PR DESCRIPTION
**Pull request by kenodegard**
# Description
Update GitHub Badge app to optionally use the GitHub PAT (it's only necessary if the repo is private).

Otherwise included a few minor spelling corrections.

# Copilot

copilot:all

### CODE REVIEW

**Comment by kenodegard** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/githubbadge/github_badge.star [hunk 1]
```diff
@@ -156,40 +157,43 @@ def main(config):
...
 
 def get_schema():
     return schema.Schema(
         version = "1",
         fields = [
             schema.Text(
                 id = "access_token",
-                name = "Github Personal Access Token",
-                desc = "Personal Access token",
+                name = "GitHub Personal Access Token",
```
Comment (Cavallando) [inline apps/githubbadge/github_badge.star:168]: I always mess this one up 😂 🙌

**Comment by Cavallando** [comment]
Very sorry for the delay @kenodegard, awesome changes, thank you very much!

@matslina looks good to me! 🚀

**Comment by matslina** [comment]
@Cavallando good idea!

**Pull request merged**